### PR TITLE
6.get/comments by review

### PR DESCRIPTION
**Pull request by BaiDom**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -115,3 +115,53 @@ describe("Get /api/reviews/:review_id", () => {
...
 });
+
+describe("GET /api/reviews/:review_id/comments", () => {
+  test("status: 200 - returns comment object that corresponds with review_id stated by clients request", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.reviews).toBeInstanceOf(Array);
+        expect(body.reviews.length).toBeGreaterThan(0);
```
Comment (yalicee) [inline __tests__/app.test.js]: Since this is test data, we just check the exact length

Comment (BaiDom) [reply __tests__/app.test.js]: cool I'll sort that out

File: __tests__/app.test.js [hunk 2]
```diff
@@ -115,3 +115,53 @@ describe("Get /api/reviews/:review_id", () => {
...
+  });
+  test("status: 404 - Path not found, if client makes request on invalid path", () => {
+    return request(app)
+      .get("/api/reviews/1/showcomments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Path not found");
+      });
+  });
+  test("status: 404 - Review not found, if client makes request on valid path but no review exists", () => {
```
Comment (yalicee) [inline __tests__/app.test.js]: this would be 400 invalid review id, not 404

**Review by yalicee — APPROVED** [review]
Approved but make the minor changes I requested :)

File: __tests__/app.test.js (hunk 2 above)
Comment (BaiDom) [reply __tests__/app.test.js]: would that be the same for all instances of an invalid review id?

**Pull request merged**